barretenberg/cpp/src/barretenberg/stdlib/recursion/verification_key/verification_key.hpp

```diff
@@ -424,8 +424,7 @@ template <typename Curve> struct verification_key {
 
         barretenberg::fr compressed_key;
         if constexpr (HasPlookup<Builder>) {
-            compressed_key = from_buffer<barretenberg::fr>(
-                crypto::pedersen_commitment::lookup::compress_native(preimage_data, hash_index));
+            compressed_key = crypto::pedersen_commitment::lookup::compress_native(preimage_data, hash_index);
```

Comment:
Not related to this PR, I think this deserves a small comment -- seeing native computation being done in the stdlib always raises suspicions for me. This same computation is also done in the non-stdlib variant of this method